src/client/Microsoft.Identity.Client/Internal/Broker/IBroker.cs

```diff
@@ -21,6 +21,10 @@ internal interface IBroker
             AuthenticationRequestParameters authenticationRequestParameters,
             AcquireTokenSilentParameters acquireTokenSilentParameters);
 
+        Task<MsalTokenResponse> AcquireTokenSilentDefaultUserAsync(
```

Comment:
>AcquireTokenSilentDefaultUserAsync [](start = 32, length = 34)

Would this be the currently signed in user on windows?

Reply:
I am not sure what the algorithm is, since you can be signed in with both a Work and School and an MSA account. But generally yes, currently signed in user.